requirements_devel.txt

```diff
@@ -13,6 +13,6 @@ mock
 pep8-naming
 pyenchant
 pylint
-pytest
+pytest==4.4
```

Comment:
do you want to >= or 4.*? or is this minor release really specifically the one?

Reply:
I only made the minor version specific due to the build failure. Maybe @r-meagher @murphm8 can way in if we need it otherwise.

Reply:
We'll make it => as discussed in the review meeting